test/utils.js

```diff
@@ -0,0 +1,48 @@
+var assert = require('chai').assert;
+var sass = require('node-sass');
+
+var render = function(options) {
+  options.outputStyle = 'compressed';
```

Comment:
I'd vote for compressed, non-overridable, with a comment explaining why it's that way.